src/apps/create-patron-user-info/CreatePatron.tsx

```diff
@@ -4,31 +4,38 @@ import { useText } from "../../core/utils/text";
 import { useConfig } from "../../core/utils/config";
 import useUserInfo from "../../core/adgangsplatformen/useUserInfo";
 
-const CreatePatron: FC = () => {
+export interface CreatePatronProps {
+  storybookContextCpr?: string;
+}
+
+const CreatePatron: FC<CreatePatronProps> = ({ storybookContextCpr }) => {
   const [cpr, setCpr] = useState<string | null>(null);
   const config = useConfig();
   const t = useText();
-
   const { id: agencyId } = config<{
     id: `${number}`;
   }>("agencyConfig", {
     transformer: "jsonParse"
   });
-  // Fetch user info data.
-  const { data: userInfo, isLoading } = useUserInfo();
+  // Fetch user info data (only if not in storybook context).
+  const { data: userInfo, isLoading } = useUserInfo({
+    enabled: !storybookContextCpr
+  });
 
   useEffect(() => {
-    if (isLoading || !userInfo) {
+    if (isLoading || !userInfo || storybookContextCpr) {
       return;
     }
-
     const {
       attributes: { cpr: userCpr }
     } = userInfo;
-
     // Otherwise set the cpr so we can show the create patron form.
     setCpr(String(userCpr));
-  }, [agencyId, isLoading, userInfo]);
+  }, [agencyId, isLoading, userInfo, storybookContextCpr]);
+
+  if (storybookContextCpr) {
```

Comment:
I think we should be very careful with mxing tools with application code. Sometimes its unavoidable, but its should always be possible to remove a tool and the remaining application code would still be running. This goes for eg. Storybook and testing.
So it seems like a small thing, but I think we can keep the prop on `<CreatePatron />` but instead call it `cpr`, make it optional and pass it to the `<UserInfo />` in line 37

Reply:
We need to call it something different than CPR because we already have a variable called that. And it indeed is optional.